config/locales/es/activerecord.yml

```diff
@@ -78,6 +81,9 @@ es:
       sdg/target:
         one: "meta"
         other: "metas"
+      sdg/manager:
+        one: "Gestor ODS"
```

Comment:
"gestor ODS" (lowercase g) seems to be more consistent with "objetivo" and "meta" :thinking:.